aioauth/server.py

```diff
@@ -215,14 +215,14 @@ def get_client_credentials(self, request: TRequest) -> Tuple[str, str]:
 
         if client_id is None or client_secret is None:
             authorization = request.headers.get("Authorization", "")
-            headers = HTTPHeaderDict({"WWW-Authenticate": "Basic"})
 
             # Get client credentials from the Authorization header.
             try:
                 client_id, client_secret = decode_auth_headers(authorization)
             except ValueError as exc:
                 raise InvalidClientError[TRequest](
-                    request=request, headers=headers
+                    description="Invalid client_id parameter value.",
```

Comment:
I'm open to other ideas here, but this was the simplest way to get all the tests to pass without having to do some funky branching for certain grant types.

Reply:
LGTM